lms/djangoapps/certificates/views/webview.py

```diff
@@ -5,15 +5,16 @@
 import logging
 import urllib
 from datetime import datetime
-import pytz
 from uuid import uuid4
 
+import pytz
```

Comment:
Let me double check. I ran isort on the files so I didn't really add the imports, it just shifted them around.